pkg/cli/cmd/cli.go

```diff
@@ -24,6 +24,7 @@ type CLI struct {
 	Manifest  ManifestCmd  `cmd:"" help:"manifest commands"`
 	Test      TestCmd      `cmd:"" help:"test assertions commands"`
 	Templates TemplateCmd  `cmd:"" help:"template commands"`
+	Certs     CertsCmd     `cmd:"" help:"cert commands"`
```

Comment:
The commands are ordered in the most common activities first, the least common/frequent ones last, and grouped by commonality (import/export, backup/restore, load/save), as such I would propose the `Certs` entry comes after `Configure`.